test/policy.bats

```diff
@@ -52,6 +53,51 @@ SANDBOX_CONFIG="$TESTDATA/sandbox_config.json"
 	assert_log "$RESTRICTIVE_POLICY"
 }
 
+@test "allow signed image if already pulled and container created during mounting phase" {
+	start_crio
+	crictl pull "$SIGNED_IMAGE"
+	stop_crio_no_clean
+
+	SIGNATURE_POLICY="$RESTRICTIVE_POLICY" start_crio
+	POD_ID=$(crictl runp "$TESTDATA/sandbox_config.json")
+	CTR_CONFIG="$TESTDIR/config.json"
+	jq --arg CONTAINER_PATH "$CONTAINER_PATH" \
+		'.image.image = "'"$SIGNED_IMAGE"'" | .image.user_specified_image = "'"$SIGNED_IMAGE"'" | .mounts = [{
```

Comment:
Let's do not mix test cases here. Can we use a standard image which will not be verified against any signature and focus on the mount part?

Reply:
I am not sure how to handle this scenario.